.gitignore

```diff
@@ -0,0 +1,3 @@
+# Editor
```

Comment:
Drop this. Add it to your global gitignore: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore